frontend/utils.mjs

```diff
@@ -241,49 +240,48 @@ export async function buildOrWatch(config) {
         }
         buildsInProgress++
         onBuildStart?.(config)
-        reloadLiveServer()
         buildPromise = runBuild()
         const buildResponse = await buildPromise
         buildPromise = null
         await onBuildComplete?.(config, buildResponse)
         buildsInProgress--
         if (buildsInProgress === 0) {
             server?.resumeServer()
+            reloadLiveServer()
```

Comment:
live reload was pretty frequently firing at the wrong time for me even before this...

let's not reload until we're done